apps/blockchain/scripts/generate_state_tests.exs

```diff
@@ -0,0 +1,2571 @@
+defmodule GenerateStateTests do
+  alias MerklePatriciaTree.Trie
+  alias Blockchain.{Account, Transaction}
+  alias Blockchain.Interface.AccountInterface
+
+  use EthCommonTest.Harness
+  # use ExUnit.Case, async: true
+
+  @ethereum_common_tests_path "../../ethereum_common_tests"
+  @passing_tests_by_group %{
```

Comment:
Is the intention for us to keep this list updated when we (and by that I mean you 😄since I'm not working on this) fix tests? If so, would it be helpful to separate this list into a file and read it in here?

Reply:
Yep, good call.

Ideally we just read the test directories and build this list dynamically. I made [an issue](https://github.com/poanetwork/mana/issues/175) to cover this in a follow-on PR.